front-end/src/renderer/stores/storeTransactionGroup.ts

```diff
@@ -100,6 +108,7 @@ const useTransactionGroupStore = defineStore('transactionGroup', () => {
    * @param index
    */
   function duplicateGroupItem(index: number) {
+    const lastItem = groupItems.value[groupItems.value.length - 1];
```

Comment:
Is there a deeper reason for adding the duplicated item to the end of the list as opposed to the adding it directly after the current item? There are potential some issues with either approach:

1. if adding the duplicate to the end, the newDate is based on the original item, and therefore the duplicate item might be added to the end but have a valid time less then previous items. 
2. if adding the duplicate just after the current item, the subsequent seq (and maybe even valid times) may need to be updated.

I think these issues might be addressed elsewhere, but should be considered. The usage of seq at all should also be considered as valid time may be all that is needed.

Reply:
After an offline conversation, we'll keep this change for the time being. This should be revisited when we're ready to discuss the need for the seq property.